server/routes/academy.js

```diff
@@ -1292,13 +1301,13 @@ router.post(
     }
 
     let classes = await network.query(networkObj, 'GetClassesByTeacher', username);
-    if (!classesQuery.success) {
+    if (!classes.success) {
       return res.status(500).json({
         success: false,
-        msg: "Can't query classes by teacher"
+        msg: "Can't query classes by teacher!"
       });
     }
```

Comment:
route này là assign teacher to class, có cần phải get classes by teacher nữa ko nhể ?

Reply:
có ông ạ, vì sau khi assign mình query lại để hiện thì luôn trong bảng của classes của teacher đó mà ông

Reply:
cái đó thì thằng front-end nó tự xử mà ông